zgw_consumers/contrib/setup_configuration/models.py

```diff
@@ -0,0 +1,36 @@
+from django_setup_configuration.models import ConfigurationModel, DjangoModelRef
+from pydantic import Field
+
+from zgw_consumers.models import Service
+
+
+class SingleServiceConfigurationModel(ConfigurationModel):
+    # TODO these should probably be defined in simple_certmanager and referred to?
+    # client_certificate: FilePath = DjangoModelRef(Service, "client_certificate")
+    # server_certificate: FilePath = DjangoModelRef(Service, "server_certificate")
+    api_connection_check_path = DjangoModelRef(
+        Service, "api_connection_check_path", default=""
```

Comment:
@swrichards not sure if there is an easier way to override the default for `CharField`s with `blank=True`